my_enumerable.rb

```diff
@@ -0,0 +1,19 @@
+module MYENUMERABLE
+  def all?()
```

Comment:
## OPTIONAL
- [ ] it's not necessary to include the parenthesis if you are not giving arguments. This does not affect your code but it makes it look cleaner.

Reply:
Yes, that's what we thought and we tried that but somehow it threw an error. I just tried it again without parenthesis and somehow it seems to be working now...